lib/exec.js

```diff
@@ -149,9 +149,23 @@ export function executeDirect(command, args, options, filenameTransform) {
     });
 }
 
+export function getNsJailCfgFilePath(configName) {
+    const configNamingMap = {
```

Comment:
we could also do something like:

`nsjail.config.{configName}=xyz` to avoid having to explicitly define a map of config options.

so:

```javascript
return execProps(`nsjail.config.${configName}`);
```
^ simplification, as we probably want to gracefully handle missing properties and show a helpful error message